src/libs/custom_logger.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+max_log_line_length = 5000
+
+
+def log_length_filter(max_length):
+    class logLengthFilter(logging.Filter):
+        def filter(self, record):
+            if len(record.getMessage()) > max_length:
+                logging.getLogger(record.name).log(
+                    record.levelno, f"Log line was discarded because it's longer than max_log_line_length={max_log_line_length}"
+                )
+                return False
+            return True
+
+    return logLengthFilter()
+
+
+def get_custom_logger(name):
```

Comment:
I just tried the `logging.config.dictConfig` now and to me my approach looks a little better as I don't have to definethe config as JSON (which increases the chanches to mess something UP)
Also there is no need for more config here because pytest contains the formatter and log level
So I would keep it as it is

Reply:
It's hard to mess it up because, as far as I recall, there's a parser/validator for the JSON. 

The only issue that was slightly bothering me was the "unnecessary" reconfiguration by repeatedly calling the method.

In any case, I'm not worried about this, I was just mainly wondering why the choice. If you feel comfortable this way then let's keep it :)